extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -96,14 +96,23 @@ export function maybeExecuteRealTimeConfig_(a4aElement, customMacros) {
   // For each vendor the publisher has specified, inflate the vendor
   // url if it exists, and send the RTC request.
   Object.keys(rtcConfig['vendors'] || []).forEach(vendor => {
-    const url = RTC_VENDORS[vendor.toLowerCase()];
+    const vendorObject = RTC_VENDORS[vendor.toLowerCase()];
+    const url = vendorObject ?
+          RTC_VENDORS[vendor.toLowerCase()].url : undefined;
     if (!url) {
       return logAndAddErrorResponse_(promiseArray,
           RTC_ERROR_ENUM.UNKNOWN_VENDOR, vendor);
     }
+    const validVendorMacros = {};
+    Object.keys(rtcConfig['vendors'][vendor]).forEach(macro => {
+      if (vendorObject.macros.includes(macro)) {
```

Comment:
do you want to allow for macros not to exist?

Reply:
ah, yea. fixed.